python/foglamp/services/core/connect.py

```diff
@@ -6,8 +6,8 @@
 
 
 from foglamp.services.core.service_registry.service_registry import ServiceRegistry
-from foglamp.common.storage_client.storage_client import StorageClient
-from foglamp.common.storage_client.storage_client import ReadingsStorageClient
+from foglamp.common.storage_client.storage_client import StorageClient, StorageClientAsync
+from foglamp.common.storage_client.storage_client import ReadingsStorageClient, ReadingsStorageClientAsync
```

Comment:
please remove StorageClient, ReadingsStorageClient to avoid confusion and errors! 
As this PR is for sync to async, let's remove StorageClient, ReadingsStorageClient completely